rock.osdeps

```diff
@@ -34,7 +34,10 @@ binding_of_caller:
     gem: binding_of_caller
 
 activesupport:
-    gem: activesupport <= 4.2.5.1
+  ubuntu: ruby-activesupport
+  debian: ruby-activesupport
+  default:
+    gem: activesupport
```

Comment:
There are no ways to deal with that, really. Autoproj 2.0 fixes it by using bundler.